contract-examples/test/tx_allow_list.ts

```diff
@@ -167,33 +157,11 @@ describe("ExampleTxAllowList", function () {
     expect.fail("should have errored")
   })
 
-
-  it("should not let allowed to revoke itself", async function () {
-    try {
-      await contract.connect(allowed).revoke(allowed.address)
-    }
-    catch (err) {
-      return
-    }
-    expect.fail("should have errored")
-  })
```

Comment:
`allowed` can't revoke anyone including itself. Blocking a non-admin is already tested.